pkg/scheduler/api/job_info.go

```diff
@@ -669,19 +669,19 @@ func (ji *JobInfo) TaskSchedulingReason(tid TaskID) (reason string, msg string)
 	case Allocated:
 		// Pod is schedulable
 		msg = fmt.Sprintf("Pod %s/%s can possibly be assigned to %s", taskInfo.Namespace, taskInfo.Name, ctx.NodeName)
-		return PodReasonSchedulable, msg
+		return PodReasonSchedulable, msg, ""
```

Comment:
`Allocated` is also a unschedulable case, we should also check whether nominatedNode is needed, maybe we can refer to kube -scheduler to deep dive.

Reply:
the autoscaler will check both condition status and reason whether a pod is unschedulable or not:
![image](https://github.com/volcano-sh/volcano/assets/26107253/9987e53c-c157-4430-8f80-f281df98065b)
so allocated will not be thought as unschedulable case to autoscaler

Reply:
Ok